### ISSUE TITLE
Support running tests on Mac OS X

### ISSUE THREAD

**Issue by xdumaine** [webrtcHacks/adapter#118]
Current output of tests on OS X:

``` text
Installing browser
+ set -e
++ dirname ./node_modules/travis-multirunner/setup.sh
+ pushd ./node_modules/travis-multirunner
++ pwd -P
+ SCRIPTPATH=/Users/xanderdumaine/Code/adapter/node_modules/travis-multirunner
+ popd
++ /Users/xanderdumaine/Code/adapter/node_modules/travis-multirunner/node_modules/.bin/browser-version chrome stable
+ TARGET_BROWSER='chrome|stable|44.0.2403.130|https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb'
++ echo 'chrome|stable|44.0.2403.130|https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb'
++ cut '-d|' -f4
+ TARGET_URL=https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
++ echo 'chrome|stable|44.0.2403.130|https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb'
++ cut '-d|' -f3
+ TARGET_VERSION=44.0.2403.130
+ TARGET_PATH=/Users/xanderdumaine/Code/adapter/node_modules/travis-multirunner/browsers/chrome/44.0.2403.130
+ BINPATH=./browsers/bin
+ mkdir -p ./browsers/bin
+ '[' '!' -d /Users/xanderdumaine/Code/adapter/node_modules/travis-multirunner/browsers/chrome/44.0.2403.130 ']'
+ echo 'getting chrome 44.0.2403.130'
getting chrome 44.0.2403.130
+ source /Users/xanderdumaine/Code/adapter/node_modules/travis-multirunner/install-chrome.sh https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb /Users/xanderdumaine/Code/adapter/node_modules/travis-multirunner/browsers/chrome/44.0.2403.130
+++ echo https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+++ sed -r 's/^.*\/([^\/]*)$/\1/'
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
++ FNAME=
Starting browser
cat: /proc/sys/kernel/random/uuid: No such file or directory
readlink: illegal option -- f
usage: readlink [-n] [file ...]
could not find chrome executable: ./browsers/bin/chrome-stable

Command undefined terminated with non-zero exit code
```

**Comment by KaptenJansson** [comment]
Current known limitations is that travis-multirunner uses .deb packages.

**Comment by KaptenJansson** [comment]
Will start looking at this soonish.

**Comment by KaptenJansson** [comment]
I believe @fippo has now solved this in #259 and #250.
